app/models/credentials.rb

```diff
@@ -3,6 +3,11 @@
 require 'bcrypt'
 require 'util/cidr'
 
+# TODO: This is needed because the loading being done in config/application.rb
+# is either not "taking" or being done after the models are loaded.
```

Comment:
I don't understand it and if we want to investigate further it needs to be a separate ticket.  But bottom line is that without this line here it acts as if these extensions aren't loaded.  Was that not clear?  If so I can just say it like that?

Reply:
Maybe then:
```
# TODO: This is needed because the loading being done in config/application.rb
#      results in extensions that do not appear to be loaded after models are processed.
```

My main point here is not about just de-obfuscating `taking` by rewording it so fell free to mod this whichever way you want.

Reply:
I want ahead and improved the comment.